test/boost/cdc_test.cc

```diff
@@ -517,15 +517,15 @@ SEASTAR_THREAD_TEST_CASE(test_primary_key_logging) {
 
 SEASTAR_THREAD_TEST_CASE(test_pre_post_image_logging) {
     do_with_cql_env_thread([](cql_test_env& e) {
-        auto test = [&e] (bool pre_enabled, bool post_enabled, bool with_ttl) {
+        auto test = [&e] (cdc::image_mode pre_enabled, bool post_enabled, bool with_ttl) {
```

Comment:
Can you please change the patch title? I don't understand what you mean by "minute test" (minute in the sense that it's small? that it runs for a whole minute? :-)), and instead of "full" perhaps say "full preimage"?

Reply:
minute == small, yes